dashboard/app/models/blockly.rb

```diff
@@ -149,14 +149,35 @@ def blockly_options
     options = Rails.cache.fetch("#{cache_key}/blockly_level_options") do
       level = self
       # Use values from properties json when available (use String keys instead of Symbols for consistency)
-      level_prop = level.properties.dup || {}
+      level_prop = {}
+
+      # Map Dashboard-style names to Blockly-style names in level object.
+      # Dashboard underscore_names mapped to Blockly lowerCamelCase, or explicit 'Dashboard:Blockly'
```

Comment:
Should this comment be updated / removed? ` or explicit 'Dashboard:Blockly'` - this portion looks like is no longer the case. maybe "or can be explicitly set in the overrides hash"